examples/config-sync.json

```diff
@@ -0,0 +1,45 @@
+{
+	"version":"0.1.0-dev",
+	"storage":{
+		"rootDirectory":"/tmp/zot"
+	},
+	"http":{
+		"address":"127.0.0.1",
+		"port":"8080"
+	},
+	"log":{
+		"level":"debug"
+	},
+	"extensions":{
+		"sync":[
+			{
+				"url":"registry1",
```

Comment:
we will need proxy information and any certs that server may publish for trust in this configuration.